File: DAY 7/Func.java
public class Func {
    static int maximumm(int[]arr){
        int maxi=arr[0];
        for(int i=0;i<arr.length;i++){
        if(maxi<arr[i]){
            maxi=arr[i];
        }
    }return maxi;
    }
    static int summ(int []arr){
        int total=0;
        for(int i=0;i<arr.length;i++){
            total+=arr[i];
        }return total;
    }
    public static void main(String[] args){
        int arr[]={12, 45, 7, 23, 56};
        int sum=summ(arr);
        int maxi=maximumm(arr);
        System.out.println("The Sum of the Array is:"+sum);
        System.out.println("The Maximum Element of the Array is:"+maxi);
    }
}

File: DAY 7/Func.c
#include<stdio.h>
int summ(int array[],int n){
    int total=0;
    for(int i=0;i<n;i++){
        total+=array[i];
    }
    return total;
}
int maximumm(int array[],int x){
    int maxi=array[0];
    for(int i=0;i<x;i++){
        if(maxi<array[i]){
            maxi=array[i];
        }
    }return maxi;
}
int main(){
    int arr[5]={12, 45, 7, 23, 56};
    int n=sizeof(arr)/sizeof(arr[0]);
    int totSum=summ(arr,n);
    int maxi=maximumm(arr,n);
    printf("The Sum of Array is:%d\n",totSum);
    printf("The Maximum Element of Array is:%d",maxi);
    return 0;
}